server/bin/pbench-audit-server.sh

```diff
@@ -713,10 +715,10 @@ function verify_archive {
     return $cnt
 }
 
-log_init $PROG
-
-# Initialize index mail content
-> ${index_content}
+# Ensure we have an existing final report file.
+report=${LOGSDIR}/${PROG}/report.latest.txt
+report_prev=${LOGSDIR}/${PROG}/report.prev.txt
+mv ${report} ${report_prev} 2>/dev/null
 > ${report}
 
```

Comment:
Does this mean we only keep the 2 most recent copies of the audit report file?

Reply:
Yes.

Basically, we just need the latest audit, since an old audit will potentially be out of date very quickly.

The previous audit is kept around so that we have it if the current audit fails.

Reply:
I have updated the code to address your concerns.